### ISSUE TITLE
Some peds failing to spawn at all

### ISSUE THREAD

**Issue by waynieoaks** [waynieoaks/RecovFR#5]
Some peds fail to spawn completely, as hash is returned.

**Comment by waynieoaks** [comment]
Hash is returned - need to build dictionary of Peds to lookup hash and match to correct name

**Comment by waynieoaks** [comment]
Blocked by issue 6: 
If the ped restore feature is not possible, this too may need to be changed to "won't fix"

https://github.com/waynieoaks/RecovFR/issues/6

**Comment by waynieoaks** [comment]
Hash dictionary completed.
